code/game/gamemodes/dynamic/dynamic_maths.dm

```diff
@@ -0,0 +1,48 @@
+// Exponential distribution.
+// Used for midround/latejoin injection delays.
+/proc/exp_distribution(var/desired_mean)
```

Comment:
This whole proc could be replaced with a define and put into the math helpers
`#define EXP_DISTRIBUTION(desired_mean) ( -(1/(1/desired_mean)) * log(rand(1, 1000) * 0.001) )`